packages/cloud-cognitive/src/components/ActionBar/ActionBar.stories.js

```diff
@@ -8,40 +8,45 @@
 import React from 'react';
 import { action } from '@storybook/addon-actions';
 
-import { Lightning16 } from '@carbon/icons-react';
+import { Bee16, Lightning16 } from '@carbon/icons-react';
 
 import { pkg } from '../../settings';
 import '../../utils/enable-all'; // must come before component is imported (directly or indirectly)
 import { getStorybookPrefix } from '../../../config';
 import { ActionBar, ActionBarItem } from '.';
-const storybookPrefix = getStorybookPrefix(pkg, ActionBarItem.displayName);
+const storybookPrefix = getStorybookPrefix(pkg, ActionBar.displayName);
 
 const blockClass = `${pkg.prefix}--action-bar`;
 
 import styles from './_storybook-styles.scss'; // import index in case more files are added later.
 
 export default {
-  title: `${storybookPrefix}/PageHeader/${ActionBarItem.displayName}`,
+  title: `${storybookPrefix}/PageHeader/${ActionBar.displayName}`,
   component: ActionBar,
   argTypes: {
     containerWidth: {
       control: { type: 'range', min: 20, max: 800, step: 10 },
     },
   },
   decorators: [
-    (story) => (
-      <div className={`${blockClass}--story__viewport`}>{story()}</div>
-    ),
+    (story) => <div className={`${blockClass}__story-viewport`}>{story()}</div>,
   ],
   parameters: { styles },
 };
 
+const actions = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10].map((num) => ({
+  renderIcon: num % 2 ? Lightning16 : Bee16,
+  iconDescription: `Action ${num}`,
+  onClick: action(`Action ${num}`),
+}));
+
 const actionBarItems = (
   <>
     <ActionBarItem
       renderIcon={Lightning16}
       iconDescription="Action 1"
       onClick={action('Action 1')}
+      data-wibble="wibble"
```

Comment:
Though I always approve of wibble in all its forms, does this line belong here?

Reply:
Removed